plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/S3BlobContainerMockClientTests.java

```diff
@@ -471,7 +473,7 @@ private void testWriteBlobByStreams(boolean expectException, boolean throwExcept
         StreamContextSupplier streamContextSupplier = partSize -> new StreamContext((partNo, size, position) -> {
             InputStream inputStream = new OffsetRangeIndexInputStream(new ByteArrayIndexInput("desc", bytes), size, position);
             openInputStreams.add(inputStream);
-            return new InputStreamContainer(inputStream, size, position);
+            return new InputStreamContainer(inputStream, size, position, null);
```

Comment:
Should we add tests to verify metadata presence or method invocations like other tests here?

Reply:
we are already verifying the method invocation count in the tests.